nilearn/_utils/niimg_conversions.py

```diff
@@ -191,9 +192,9 @@ def check_niimg(niimg, ensure_ndim=None, atleast_4d=False, dtype=None,
     ----------
     niimg : Niimg-like object
         See https://nilearn.github.io/stable/manipulating_images/input_output.html  # noqa:E501
-        If niimg is a string, consider it as a path to Nifti image and
-        call nibabel.load on it. The '~' symbol is expanded to the user home
-        folder.
+        If niimg is a string or path-like object, consider it as a path to
```

Comment:
I'm wondering whether we should create a standard doc entry for `niimg` ? WDYT?

Reply:
I was actually thinking the same thing!

Reply:
On second thought, is this applicable in the `_utils` module given that it's private? I haven't seen this standardization in any of the other files inside this module (but I didn't look too extensively). So I was wondering if this is by choice or just hasn't been done yet.

Reply:
The docstrings in the `_utils` module are indeed not rendered since it is private, but for docstrings of public functions, that could still be useful.

Reply:
Ok I will hold off to doing that in this PR then but I agree it would be useful.